### ISSUE TITLE
Keep builds up to date

### ISSUE THREAD

**Issue by birm** [msalsbery/OpenSeadragonImagingHelper#13]
The readme points to a build from 2015-01-08 (http://msalsbery.github.io/builds/openseadragonimaging/openseadragon-imaginghelper.min.js)
This link should either point to a more up to date build or be removed, I think.

**Comment by iangilman** [comment]
@msalsbery What's involved in doing a build? Do you have it documented somewhere? Is it just a matter of doing `grunt publish` in the `gh-pages` branch?

**Comment by msalsbery** [comment]
@iangilman Just grunt or grunt build should do it. I believe grunt publish only copies the build to the appropriate demo site folder

**Comment by iangilman** [comment]
@msalsbery I see... Looks like there's no `gh-pages` on this project; it's hosted on https://github.com/msalsbery/msalsbery.github.io instead. I don't have access to that project, so you'll have to publish there (or give me access to that one too).

**Comment by msalsbery** [comment]
@iangilman There's a couple invites on the way to you. Thank you!

**Comment by iangilman** [comment]
@msalsbery Thanks! 

Okay, I've updated the built files: 

http://msalsbery.github.io/builds/openseadragonimaging/openseadragon-imaginghelper.js
http://msalsbery.github.io/builds/openseadragonimaging/openseadragon-imaginghelper.min.js

I wanted to update the docs as well, but I ran into trouble... It keeps complaining about modules it can't find, and when I `npm install` those modules it complains about the next one. I'm finally stuck here, since I've already installed `underscore`:

```
Running "jsdoc:dist" (jsdoc) task
>> An error occurs in jsdoc process:
>> js: "/Users/iangilman/projects/openseadragon/OpenSeadragonImagingHelper/node_modules/grunt-jsdoc/node_modules/jsdoc/jsdoc.js", line 316: exception from uncaught JavaScript throw: Error: Module "underscore" not found.
Warning: jsdoc failure Use --force to continue.
```

I guess something is just broken with jsdoc at the moment? Anyway, at least we have the code files updated.
